README.md

```diff
@@ -12,11 +13,11 @@ Currently, this package supports the following ad sources:
 
 ## Models
 
-This package contains a number of models, all building up to the `ad_reporting` model, which is a combination of all the data sources. A dependency on all the required dbt packages is declared in this package's `packages.yml` file, so it will automatically download them when you run `dbt deps`. The primary outputs of this package are described below.
+This package contains a number of models, which all build up to the `ad_reporting` model. The `ad_reporting` model combines the data from all of the data sources. A dependency on all the required dbt packages is declared in this package's `packages.yml` file, so it will automatically download them when you run `dbt deps`. The primary outputs of this package are described below.
```

Comment:
What do you mean by "build up"? "Building up" usually means "leading up to," which doesn't make sense to me in this context.

Reply:
Also, from the Models section onward, we use "data source" instead of "connector." Is the data at the connector level or at the source level? For example, if the customer has 2 Pinterest Ads connectors, will they be able to distinguish between the two in the final data set?

Reply:
Build up in this context means that they are downstream models that are necessary to create the final `ad_reporting` model.  Essentially they are intermediate models that transform all the source data in a way that can then be used by our final model.

Reply:
Good catch - we should use "connector" rather than "data source"